website/docs/r/azurerm_postgresql_flexible_server_virtual_endpoint.html.markdown

```diff
@@ -0,0 +1,93 @@
+  location                      = azurerm_resource_group.example.location
+  version                       = "16"
+  public_network_access_enabled = false
+  administrator_login           = "psqladmin"
+  administrator_password        = "H@Sh1CoR3!"
+  zone                          = "1"
+
+  storage_mb   = 32768
+  storage_tier = "P30"
+
+  sku_name = "GP_Standard_D2ads_v5"
+}
+
+resource "azurerm_postgresql_flexible_server" "example_replica" {
+  name                          = "example-replica"
+  resource_group_name           = azurerm_postgresql_flexible_server.example.resource_group_name
+  location                      = azurerm_postgresql_flexible_server.example.location
+  create_mode                   = "Replica"
+  source_server_id              = azurerm_postgresql_flexible_server.example.id
+  version                       = "16"
+  public_network_access_enabled = false
+  zone                          = "1"
+  storage_mb                    = 32768
+  storage_tier                  = "P30"
+
+  sku_name = "GP_Standard_D2ads_v5"
+}
+
+resource "azurerm_postgresql_flexible_server_virtual_endpoint" "example" {
+  name              = "example-endpoint-1"
+  source_server_id  = azurerm_postgresql_flexible_server.example.id
+  replica_server_id = azurerm_postgresql_flexible_server.example_replica.id
+  type              = "ReadWrite"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Virtual Endpoint
+
+* `source_server_id` - (Required) The Resource ID of the *Source* Postgres Flexible Server this should be associated with.
+
+* `replica_server_id` - (Required) The Resource ID of the *Replica* Postgres Flexible Server this should be associated with
+
+* `type` - (Required) The type of Virtual Endpoint. Currently only `ReadWrite` is supported.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported:
+
+* `id` - The ID of the PostgreSQL Flexible Virtual Endpoint.
+
+## Timeouts
+
+The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/language/resources/syntax#operation-timeouts) for certain actions:
+
+* `create` - (Defaults to 30 minutes) Used when creating the PostgreSQL Flexible Virtual Endpoint.
+* `update` - (Defaults to 30 minutes) Used when updating the PostgreSQL Flexible Virtual Endpoint.
+* `read` - (Defaults to 5 minutes) Used when retrieving the PostgreSQL Flexible Virtual Endpoint.
+* `delete` - (Defaults to 30 minutes) Used when deleting the PostgreSQL Flexible Virtual Endpoint.
+
+## Import
+
+A PostgreSQL Flexible Virtual Endpoint can be imported using the `resource id`, e.g.
+
+```shell
+terraform import azurerm_postgresql_flexible_server_virtual_endpoint.example subscriptions/<subscriptionId>/resourceGroups/<resourceGroupName>/providers/Microsoft.DBForPostgreSql/flexibleServers/<sourceserverName>/virtualendpoints/<virtualendpointName>
```

Comment:
I don't believe I've seen us use `<userSuppliedSegmentHere>` notation in the import command for the docs before, purely for consistency can we update this to be consistent with the rest of the docs?
```suggestion
terraform import azurerm_postgresql_flexible_server_virtual_endpoint.example subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/group1/providers/Microsoft.DBForPostgreSql/flexibleServers/server1/virtualendpoints/endpoint1
```